lte/gateway/python/load_tests/README.md

```diff
@@ -16,34 +16,25 @@ failure, and so on. The options for configuration and such can be looked
 
 ### Gateway VM setup
 
-Spin up and provision the gateway VM, then make and start its services:
+Spin up and provision the gateway VM. From `magma/lte/gateway` on the host machine run `vagrant up magma && vagrant ssh magma`.
 
-1. From `magma/lte/gateway` on the host machine:
-   `vagrant up magma && vagrant ssh magma`
-2. Now in the gateway VM: `cd $MAGMA_ROOT/lte/gateway && make run`
+Next build and start the magma services. See the [integ test documentation](https://docs.magmacore.org/docs/next/lte/s1ap_tests#s1ap-integration-tests) for details.
 
 ### Run tests
 
-From `$MAGMA_ROOT/lte/gateway/python/load_tests` on the *magma* VM, run:
-
-* All tests: `make load_test`
+On the gateway VM, run `$MAGMA_ROOT/bazel/scripts/run_load_tests.sh` from anywhere inside the magma folder.
```

Comment:
I always get an error when I try to execute this (no matter if I built magma with make or bazel):
```
INFO: Elapsed time: 12.587s, Critical Path: 1.85s
INFO: 1671 processes: 1629 disk cache hit, 13 internal, 29 linux-sandbox.
INFO: Build completed successfully, 1671 total actions
Finished building load tests.
#############################
Running load tests...
#############################
Running load test: loadtest_mobilityd allocate
sudo: /home/vagrant/magma/bazel-bin/lte/gateway/python/load_tests/loadtest_mobilityd allocate: command not found
```

Maybe there is a problem running Bazel while you're in the virtualenv? I just restarted a build...

Reply:
Sorry about that, I was just fixing a shellcheck warning which caused this. I fixed it - see comment in the script.

Reply:
The issue was that it was looking for the file `"home/vagrant/magma/bazel-bin/lte/gateway/python/load_tests/loadtest_mobilityd allocate"` which does not exist with the ` allocate` in the same string.